### ISSUE TITLE
Remove incorrect parent - DNA replication, synthesis of RNA primer

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#22572]
GO:0006269 DNA replication, synthesis of RNA primer
is_a DNA metabolic process; but DNA is only used as a template , not metabolized. I will remove this relation. 

Found during design pattern discussion with @cmungall and @balhoff 

Thanks, Pascale

**Comment by ValWood** [comment]
This term is the reason why "DNA replication" cannot go under "DNA metabolic process". I always find that missing parent a bit weird. The RNA primer is only an intermediate,  which is later removed. Do you think there could be an alternative more biologically intuitive way to model this?

**Comment by pgaudet** [comment]
Yes makes sense

**Comment by pgaudet** [comment]
I removed 

GO:0034645 ! cellular macromolecule biosynthetic process as a direct parent of 'DNA replication'. That leaves the logical definition: 
 intersection_of: GO:0009058 ! biosynthetic process
 intersection_of: has_primary_output CHEBI:16991 ! deoxyribonucleic acid

which was already there. This infers that DNA replication is_a DNA metabolic process. 

---

Also, I changed the labels of: 
GO:0006261 DNA-dependent DNA replication
to  DNA-templated DNA replication

and 
RNA-dependent DNA biosynthetic process

and all related regulation terms.
